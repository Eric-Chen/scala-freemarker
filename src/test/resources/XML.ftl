[#ftl]
<ul>
[#list root.child[0]?children as c]
  <li id="${c.@id}">
    <p>${c?node_name}</p>
    <p>${c?node_type}</p>
    <p>${c}</p>
  </li>
[/#list]
</ul>