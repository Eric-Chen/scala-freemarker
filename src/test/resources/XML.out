<ul>
  <li id="0">
    <p>one</p>
    <p>element</p>
    <p>1</p>
  </li>
  <li id="1">
    <p>two</p>
    <p>element</p>
    <p></p>
  </li>
  <li id="2">
    <p>three</p>
    <p>element</p>
    <p>Three</p>
  </li>
</ul>